Tests/LibWeb/Text/input/HTML/Navigation-object-properties.html

```diff
@@ -11,6 +11,15 @@
 
         println(`transition is null: ${n.transition == null}`);
         println(`canGoBack: ${n.canGoBack}`);
```

Comment:
I have no idea why our test runner causes this to become false due to this change, but I can't say I think it should have been true to begin with.